privacy_bot/analysis/utils.py

```diff
@@ -8,6 +8,9 @@
 
 
 class Dataset():
```

Comment:
Shouldn't `url dataset` and `policies dataset` be two different classes? Or at least be loaded by two different functions? It would look more 'scikit-like' to me. (judging by what they propose there: http://scikit-learn.org/stable/datasets/)